Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ however, it has to be formatted properly to pass verification tests.
 
 ### Changed
 
+- Changed `TrackedPoseDriver` to use properties of type `InputActionProperty` rather than `InputAction` to allow more flexibility.
+- XRLayoutBuilder now supports `_` in sanitized names.
```

Comment:
Sidenote: neither of these is particularly helpful/useful.

The first entry would ideally highlight how this increased flexibility and what in particular was impacted. The second entry mentions an internal API that is meaningless to the user. Likely, the `_ in sanitized names` part probably doesn't mean anything to user.